backend/ibutsu_server/db/models.py

```diff
@@ -238,12 +238,19 @@ def password(self):
         return self._password
 
     @password.setter
-    def password_set(self, value):
+    def password_set(self, value: str):
```

Comment:
You might want to make this property use the `set_password` method below, so that the password hash generation happens in only one place. This will prevent future frustration when someone accidentally only changes one but not the other.